aspnetcore/fundamentals/localization.md

```diff
@@ -257,24 +257,24 @@ Suppose you want to let your customers store their language and culture in your
 
 ```csharp
 services.Configure<RequestLocalizationOptions>(options =>
-   {
-       var supportedCultures = new[]
-       {
-           new CultureInfo("en-US"),
-           new CultureInfo("fr")
-       };
-
-       options.DefaultRequestCulture = new RequestCulture(culture: "en-US", uiCulture: "en-US");
-       options.SupportedCultures = supportedCultures;
-       options.SupportedUICultures = supportedCultures;
-
-       options.RequestCultureProviders.Insert(0, new CustomRequestCultureProvider(async context =>
-       {
-         // My custom request culture logic
-         return new ProviderCultureResult("en");
-       }));
-   });
-   ```
+{
+    var supportedCultures = new[]
+    {
+        new CultureInfo("en-US"),
```

Comment:
I suggest storing "en-US" in a constant, since it's referenced 3 times in this snippet.